### ISSUE TITLE
UI suggestion

### ISSUE THREAD

**Issue by pooyaww** [SonyaCore/V2RayGen#5]
Hi,
How do you think about adding X-UI panel to your script so that the admin can manage the server and accounts easier.

**Comment by SonyaCore** [comment]
I can add an argument for installing x-ui 
but the main goal of this script is to setup v2ray in the fastest way.

https://seakfind.github.io/2021/10/10/X-UI/
the x-ui requires ssl & domain, without that trojan & tls options become pointless.

**Comment by pooyaww** [comment]
@SonyaCore 
I agree that simplicity and lightweightness are essential features for this script.
I have installed the X-UI on a server after running V2RayGen without any ssl and domain and it works fine.
However, I don't know anything about Trojan and TLS.
The benefits for me is being able to make unique accounts for different users with different configurations and also checking my instance's total data transmission and bandwidth.
It is the English version of X-UI that I used [https://github.com/proxykingdev/x-ui](https://github.com/proxykingdev/x-ui) .
I also think adding it only as an optional argument makes sense :)
